tools/fio-engine/xnvme_fioe.c

```diff
@@ -189,6 +190,15 @@ static struct fio_option options[] = {
 		.category = FIO_OPT_C_ENGINE,
 		.group = FIO_OPT_G_INVALID,
 	},
+	{
+		.name   = "xnvme_be",
+		.lname  = "xNVMe Backend",
+		.type   = FIO_OPT_STR_STORE,
+		.off1   = offsetof(struct xnvme_fioe_options, xnvme_be),
+		.help   = "Select backend. cmd-interface: []",
```

Comment:
See the other comment, you remove this in the other commit, just don't introduce it here :)